lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParserClang.cpp

```diff
@@ -519,6 +519,33 @@ TypeSP DWARFASTParserClang::ParseTypeFromDWARF(const SymbolContext &sc,
   return UpdateSymbolContextScopeForType(sc, die, type_sp);
 }
 
+static std::optional<uint32_t>
```

Comment:
The integer types are a bit inconsistent (but this didn't change with this patch).

* `ParseInheritance` stores the value in an `off_t`. Then passes it to `clang::CharUnits::fromQuantity` which takes an `int64_t`
* The Rust `Variant` support stores offsets as uint32_t, and so does `MemberAttributes`. But `FormValue::Unsigned()` returns a uint64_t.

Would be nice to make this consistent eventually